firmware/applications/SignBuddy/sensors.h

```diff
@@ -7,4 +7,6 @@ void sensors_task_setup(void);
 void sensors_task_start(void);
 void sensors_sampling_timer_start(void);
 
+void sensors_sample_periodic(uint8_t);
```

Comment:
at least use bool instead of uint8_t 

but i think this is a bad way to indicate that you want to sample once if it's 0. 

it's quite confusing.

why not have `sensors_sample` and then pass an enum, for period or static

Reply:
Yeah I'll change it

Reply:
Thanks for this suggestion, it makes way more sense to do it this way